<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetCalculationResultsDialog</class>
 <widget class="QDialog" name="GetCalculationResultsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GetCalculationResultsDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="FileNameLabel">
         <property name="text">
          <string>文件：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="FileNameLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="InputFilePushButton">
         <property name="text">
          <string>输入文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="GetResultPushButton">
         <property name="text">
          <string>获取结果</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="ResultLabel">
           <property name="text">
            <string>结果</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="ResultTextEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="SaveAspushButton">
         <property name="text">
          <string>另存为</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PrintPushButton">
         <property name="text">
          <string>打印</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
